### ISSUE TITLE
[Package Request] nZEDb

### ISSUE THREAD

**Issue by sirjaymz** [SynoCommunity/spksrc#2356]
Description: nZEDb
Website: https://github.com/nZEDb/nZEDb
Documentation: https://github.com/nZEDb/nZEDb/wiki
Build/Installation documentation:  
Source Code:  https://github.com/nZEDb/nZEDb/
License:  GPL v3

will submit bounty as well. looking to move off of newznab and onto a more community supported solution on my DS 1813+.

**Comment by sirjaymz** [comment]
https://www.bountysource.com/issues/34773881-package-request-nzedb

**Comment by Dr-Bean** [comment]
@Diaoul Did anything change with the BountySource integration? Seems it's gone since a month or so.

**Comment by Diaoul** [comment]
I don't know, it should still work, spksrc is marked as active.

**Comment by cytec** [comment]
ha! i knew that would pop up sooner or later for reference https://github.com/SynoCommunity/spksrc/issues/2256 maybe i'll get that done :)

**Comment by Dr-Bean** [comment]
@Diaoul FYI, we're not the only ones: https://github.com/bountysource/core/issues/1018.

**Comment by schumi2004** [comment]
I noticed @cytec working on a package for this but he lowered version to 0.5.4 and i think that's because in more recent builds composer is needed to install nZEDb
Therefore i tried to do a manual install of nZEDb but it was missing certain PHP extensions and i couldn't get it running (getting composer.phar went pretty straight forward)

https://github.com/cytec/spksrc/tree/nzedb

**Comment by cytec** [comment]
yeah i stopped working on that a while ago... there where more and more errors popping up... the composer stuff isn't too hard to handle but the install (mostly the initial database filling doesn't work correctly) which seems to be a general problem of the nZEDb installer atm

**Comment by schumi2004** [comment]
@cytec could you still have a look at nZEDb package?
I compiled it (version 0.5.5) and ran setup but it complained about PEAR being missing.
So i added cross/pear to Makefile and compiled again but now i'm getting this: http://pastebin.com/93wSHTbk
I didn't touched a button when that popped-up so i have no clue what to do.
Also check cross/pear/Makefile but all seems good to me.

Thanks in advance.

**Comment by hgy59** [comment]
nZEDb is not actively maintained anymore.
